packages/icons-scripts/scripts/reactify.js

```diff
@@ -2,7 +2,7 @@ const Compiler = require('svg-baker');
 
 const compiler = new Compiler();
 
-const reactify = (symbol, componentName) => {
+const reactify = (symbol, componentName, deprecated, replaceWithComponentName) => {
   const width = symbol.viewBox.split(' ')[2];
   const height = symbol.viewBox.split(' ')[3];
 
```

Comment:
jsdoc бы добавить, по типу

```suggestion

  let jsdoc = ""
  if (deprecated) {
    const replacementNotice = replaceWithComponentName
      ? `. Замените на ${replaceWithComponentName}`
      : "";

    jsdoc = `/**
 * @deprecated Иконка устарела${replacementNotice}
 */
`
  }

```